Lombiq.VueJs/Assets/Scripts/helpers/rollup-pipeline.js

```diff
@@ -1,72 +1,111 @@
-const all = require('gulp-all');
+const fs = require('fs');
 const path = require('path');
-const gulp = require('gulp');
-const plumber = require('gulp-plumber');
-const rollup = require('gulp-rollup');
-const rename = require('gulp-rename');
-const uglify = require('gulp-uglify');
-const log = require('fancy-log');
+const { minify } = require('terser');
+const { rollup } = require('rollup');
+
+const { handleErrorObject } = require('nodejs-extensions/scripts/handle-error');
+
+function createDirectory(directoryPath) {
+    return fs.existsSync(directoryPath) ? Promise.resolve() : fs.promises.mkdir(directoryPath);
+}
+
+function handleRollupError(error) {
+    const stack = error?.stack?.split(/[\n\r]+/).map((line) => line.trim()) ?? [];
+    const extensionRegex = /\.(js|vue):(\d+)(:(\d+))?.*/;
+    const pathLine = stack
+        .filter((line) => line.match(extensionRegex))
+        .map((line) => line.replace(/^\s*at\s+/, '').trim())[0];
+    const lineMatch = pathLine?.match(extensionRegex) ?? [];
+
+    handleErrorObject({
+        code: 'ROLLUP',
+        path: pathLine?.replace(extensionRegex, '.js'),
+        line: lineMatch ? lineMatch[2] : undefined,
+        column: lineMatch ? lineMatch[4] : undefined,
```

Comment:
Using named groups, as suggested above, this becomes:
```suggestion
        line: lineMatch ? lineMatch.groups.line : undefined,
        column: lineMatch ? lineMatch.groups.column : undefined,
```